assets/scss/main.scss

```diff
@@ -1172,8 +1172,8 @@ body {
                 left: 0;
                 content: ' ';
                 background-color: #E5F1FC;
-                z-index: -1;
                 width: 100%;
+                z-index: -1;
```

Comment:
Is he switch needed?

Reply:
maybe not, it is inherited from father anyway, but we can fix it in a dedicated issue/pr

Reply:
@sebbalex Did you already create the new issue for this?